libs/scripts/scripts.js

```diff
@@ -44,7 +44,15 @@ export async function loadLCP(blocks) {
   }
 }
 
-function loadDelayed() {}
+/**
+ * Load all the delayed things. This code is temporary for testing purposes and will be removed after code review and QA.
+ */
+function loadDelayed() {
```

Comment:
I think we put this in delayed.js and actually delay it by 3000 like our consumers. Business Blog & Blog should have samples of this.

Reply:
I will move and adjust this.  I know exactly where you are talking.